src/Nancy/Conventions/BuiltInCultureConventions.cs

```diff
@@ -52,8 +41,9 @@ public static CultureInfo FormCulture(NancyContext context)
         /// Checks to see if the first argument in the Path can be used to make a CultureInfo.
         /// </summary>
         /// <param name="context">NancyContext</param>
+        /// <param name="configuration">Culture configuration that contains allowed cultures</param>
         /// <returns>CultureInfo if found in Path otherwise null</returns>
-        public static CultureInfo PathCulture(NancyContext context)
+        public static CultureInfo PathCulture(NancyContext context, GlobalizationConfiguration configuration)

```

Comment:
In two minds about passing in the entire configuration class vs just the supported culture names. But maybe it's fine